### PR TITLE
Add version history for 5.1.2

### PR DESCRIPTION
**Pull request by melissalinkert**
As usual, every single bug fix is not listed, but I think this catches the notable ones.  Happy to add/remove if needed.

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,33 @@
 Version history
 ===============
 
+5.1.2 (2015 May 28)
+-------------------
+
+* Added OME-TIFF writing support to the native C++ implementation
+* OME-TIFF export: switch to BigTIFF if .ome.tf2, .ome.tf8, or .ome.btf
+  extensions are used
+* Improved to MATLAB developer documentation
```
Comment (joshmoore) [inline docs/sphinx/about/whats-new.txt]: "Improvements to" or just "Improved"

**Comment by hflynn** [comment]
Barring Josh's comment, this reads and renders fine on staging so good to go once that sentence is fixed.

**Comment by melissalinkert** [comment]
Fix pushed.

**Comment by hflynn** [comment]
:+1: Good to merge

**Pull request merged**